### ISSUE TITLE
Regular DNS stamp for relay servers

### ISSUE THREAD

**Issue by junkurihara** [DNSCrypt/dnscrypt-proxy#1558]
## *When* does this occur?
Unlike a short 'sdns://...' of a dns stamp including only an IP address and a port, when we set a 'regular' dns stamp including full information (much like ones for Dnscrypt resolvers)  for relay servers (acting as resolvers) in a configuration file, it doesn't get loaded and such a relay server with full dns stamp are not set as relay servers.

## *How* do we replicate the issue?

1. Setup your DNSCrypt relay server or get a 'full' dns stamp of an existing relay server
2. Put the 'full' dns stamp of a relay to your configuration file like
```
routes = [
  { server_name = '*', via = [
    'sdns://......................................................................',
  ] },
]
```

## Expected behavior (i.e. solution)

In the documantation (https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Anonymized-DNS), the following was mentioned clearly. So it should work for regular and full dns stamps.

> Relay names can also be replaced with DNS stamps (sdns://...), IPs and ports (ip:port) or hostnames and ports ('host:port). If a resolver also acts as a relay, it's possible to use its regular stamp instead of the relay stamp.

## Other Comments

Here's my commit resolving this issue. If you like i'll make a PR.

https://github.com/junkurihara/dnscrypt-proxy/commit/81cb540babff9db7bf0fb69b9098f08392ffc360

**Comment by jedisct1** [comment]
Good catch, thanks!

**Comment by junkurihara** [comment]
Ok! I'll make a PR!

**Comment by jedisct1** [comment]
It was fixed in  254a4a6 :)
